docs/proposals/componentconfig-dump.md

```diff
@@ -0,0 +1,101 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should refer to the docs that go with that version.
+
+Documentation for other releases can be found at [releases.K8s.io](http://releases.K8s.io).
+</strong>
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+
+# Overview
+
+We describe a proposal for retrieving component configurations (componentconfigs) that govern the behavior of any one of the Kubernetes (K8s) components. 
+
+We propose several important changes:
+* adding new authenticated protected path to a component api: 
+    * `/componentconfig` 
```

Comment:
Just FYI: The Kubelet presently exposes the config it thinks it has via /configz, but that's more for debug/checking from tests.